### ISSUE TITLE
Create quiet-profiles.json readme

### ISSUE THREAD

**Issue by brian-armstrong** [brian-armstrong/quiet#2]
Some documentation about quiet-profiles.json would be handy both for quiet and quiet-js. Documentation already exists about the structs that the profiles generate so this process should mostly be a matter of extracting that information and exposing it in a new document.

**Comment by nicolas-f** [comment]
Hi,

Yes it would be helpful. We have some difficulties to transmit data over sound. I would like to reduce the bandwidth but be more robust to background noise and distortions (reverberation and resonance).

I have no clue to how to achieve that.

Thanks

**Comment by brian-armstrong** [comment]
Hi Nicolas,

I have actually been working on better documentation recently, but there's still work left before it's ready.

This might be helpful if you're trying to get a better intuition about profiles https://quiet.github.io/quiet-profile-lab

Unfortunately the lab only supports interpolation factors (samples_per_symbol) up to 64, but the modem itself can go even higher. Using the standard modem in BPSK or OOK mode with an interpolation of at least 50 may help you achieve what youre looking to do

**Comment by nicolas-f** [comment]
Thanks for your answer it helps a lot 👍

**Comment by goblin** [comment]
Agreed, more documentation would be super-nice. The profile lab doesn't seem to work in my browser (Chromium). After I press "start lab" I get "Sorry, it looks like this example is not supported by your browser. Please give permission to use the microphone or try again in Google Chrome or Microsoft Edge." (and the browser is not asking for any permissions).

**Comment by brian-armstrong** [comment]
I'm not able to repro the issue you mentioned, and I just tried with Chome 74. Can you refresh and try it again? And for follow ups please move discussion to https://github.com/quiet/quiet-profile-lab/issues

**Comment by b-d055** [comment]
@brian-armstrong do you have more details or an example of how to use OOK? No mater what options I put in it seems the modulation/output doesn't change. 

I don't see ook mentioned anywhere in the documentation. (I'm actually using quiet-js but I've been looking at quietlib since the code explains a lot)

@nicolas-f Did you ever get an OOK profile working?
